backend/app/domains/grafana/routers.py

```
@@ -0,0 +1,32 @@
+@router.get("/cpu-usage", status_code=HTTP_200_OK)
+async def get_cpu_usage():
+    return get_cpu(1.0)
+
+
+@router.get("/memory-usage", status_code=HTTP_200_OK)
+async def get_memory_usage():
+    return get_memory()
+
+
+@router.get("/pod-status", status_code=HTTP_200_OK)
+async def get_pod_status():
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)